### PR TITLE
imported wiki page 'live images'

### PR DESCRIPTION
**Pull request by bobertlo**
This is basically the entire 'Live Images' wiki page. I didn't put the account credentials in yet because they will probably go into the install guide. I didn't get fancy with links or highlights, in an effort to merge the entire page.

### CODE REVIEW

File: src/installation/liveimages/baserequirements.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Base Image System Requirements
+
+These are the minimum system requirements when using a live base image (without a bundled Desktop Environment)
```
Comment (the-maldridge) [inline src/installation/liveimages/baserequirements.md]: Lines that aren't part of some kind of syntactic structure (e.g. a table) should be wrapped at 80 columns.

File: src/installation/liveimages/flavours.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+|Terminal|Terminology|gnome-terminal|LXTerminal|QTerminal|MATE terminal|xfce4-Terminal|
+|Document Viewer|-|-|-|-|Atril (PS/PDF)|-|
+|Plain text viewer|-|-|-|-|Pluma|Mousepad|
+|Image viewer|-|-|GPicView|LXImage|Eye of MATE|Ristretto|
+|Archive unpacker|-|-|-|-|Engrampa|-|
+|Other|Mixer, EConnMan (connection manager), Elementary Test|-|LXTask (task manager), MIME type editor|Screen grabber|Screen grabber, file finder, MATE color picker, MATE font viewer, Disk usage analyzer, Power statistics, System monitor (task manager), Dictionary, Log file viewer|Bulk rename, Orage Globaltime, Orage Calendar, Task Manager, Parole Media Player, Audio Mixer, MIME type editor, Application finder|
+
+Please note that the system requirements for these graphical live images are higher and will vary depending on which desktop environment you choose. Also, currently some images can be problematic:
+
+ * The ''musl'' LXQt image comes with a [broken web browser](https://github.com/voidlinux/void-mklive/issues/120)
```
Comment (the-maldridge) [inline src/installation/liveimages/flavours.md]: Outdated, please remove.

File: src/installation/liveimages/flavours.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ * The ''musl'' Cinnamon image comes with a [broken DE](https://forum.voidlinux.eu/t/x86-64-iso-musl-cinnamon-segfaults-unable-to-use/2172)
```
Comment (the-maldridge) [inline src/installation/liveimages/flavours.md]: Outdated, please remove.

File: src/installation/liveimages/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Live Installer Images
+
+Live Insaller images contain a base set of utilities as well as an installer to install a Void system.
```
Comment (the-maldridge) [inline src/installation/liveimages/index.md]: installer

File: src/installation/liveimages/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Void provides glibc based live images for both i686 and x86_64, as well as musl based live images for x86_64-compatible processors.
+Each platform provides a base install image as well as ready-to-try desktop environment images.
+
+All live images and rootfs tarballs are available at [https://repo.voidlinux.eu/live/current/](https://repo.voidlinux.eu/live/current/)
```
Comment (the-maldridge) [inline src/installation/liveimages/index.md]: https://alpha.de.repo.voidlinux.org/live/current

**Comment by CameronNemo** [comment]
This repeats some information already in the downloading page. I feel like the only valuable information here is the resource requirements and flavors comparison.

**Comment by the-maldridge** [comment]
I'm inclined to agree with @CameronNemo that there's a lot of duplication here.  Also we're in lang en_US for the rest of the docs, if anyone wants to do a translation later we can investigate alternate spellings of common words.

**Comment by CameronNemo** [comment]
@bobertlo that sounds like a good plan. Thank you for contributing!

**Comment by bobertlo** [comment]
Ok, I did a lot of work merging all the information together in a reasonable flow. I checked all the URLs and updated the information on gpg signatures (assuming we don't need to mention xtraeme's key for now.) Since all the download/media creation/requirements pages are only concerning the live images I put them in that section.

**Comment by bobertlo** [comment]
I do still see a reason to generalize downloading/verifying and media creation into a higher level since the images do apply to embedded systems as well. I would just like to get this pulled since there are some important changes regarding the repo URLs and signatures, and then we can move on to cover other install procedures as needed in smaller pieces.

File: src/installation/liveimages/downloading.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+maintainers published and signed you can use pgp.
+
+The file is signed with the Void Images key:
+* Signer: Void Linux Image Signing Key <<images@voidlinux.eu>>
+* KeyID: ``B48282A4``
+* Fingerprint: ``CF24 B9C0 3809 7D8A 4495 8E2C 8DEB DA68 B482 82A4``
+
+You can use [gpg(1)](https://man.voidlinux.org/gpg.1) to receive the
+key from a keyserver using the following command or download it from
+[https://alpba.de.repo.voidlinux.eu/live/current/void_images.asc](https://alpha.de.repo.voidlinux.org/live/current/void_images.asc).
```
Comment (CameronNemo) [inline src/installation/liveimages/downloading.md]: typo here, should be alpha

Comment (bobertlo) [reply src/installation/liveimages/downloading.md]: Hah, I went and clicked all the links to double check. I typed that URL many times last night...

**Comment by CameronNemo** [comment]
You could squash the typo commit into the other commit, but the content looks good to me!

**Comment by bobertlo** [comment]
Thanks! The other night I was wary of making too many changes from the wiki since I am very new to void.

**Pull request merged**